lib/components/authentication/widgets/login_screen.dart

```diff
@@ -87,21 +91,34 @@ class _LoginScreenState extends State<LoginScreen> {
     return SlideAnimation(
       key: _slideLoginKey,
       endPosition: Offset(0, -mediaQuery.size.height),
-      child: Column(
-        children: <Widget>[
-          Expanded(child: _buildText()),
-          const SizedBox(height: 16),
-          Expanded(
-            flex: 2,
-            child: Column(
-              children: <Widget>[
-                Expanded(child: _buildTitle(theme)),
-                Expanded(child: _buildLogo()),
-              ],
+      child: SafeArea(
+        child: Column(
+          children: <Widget>[
+            Align(
+              alignment: Alignment.centerRight,
+              child: HarpyButton.flat(
+                icon: const Icon(FeatherIcons.info),
+                padding: const EdgeInsets.all(16),
+                onTap: () async {
+                  await app<HarpyNavigator>().state.maybePop();
+                  app<HarpyNavigator>().pushNamed(AboutScreen.route);
+                },
```

Comment:
Can be changed to `onTap: () => app<HarpyNavigator>().pushNamed(AboutScreen.route)`